demisto_sdk/commands/common/hook_validations/structure.py

```diff
@@ -143,12 +146,10 @@ def is_valid_scheme(self):
             core.validate(raise_exception=True)
         except Exception as err:
             try:
-                error_message, error_code = self.parse_error_msg(err)
-                if self.handle_error(error_message, error_code, self.file_path,
-                                     suggested_fix=Errors.suggest_fix(self.file_path)):
-                    self.is_valid = False
-                    return False
-            except Exception:
+                return self.parse_error_msg(err)
+            except Exception as e:
+                print('error')
+                print(e)
```

Comment:
leftovers from testings?
if not, why aren't you using `click`?

Reply:
leftovers... thanks